tutorials/CRPS_for_Ensembles.ipynb

```diff
@@ -25,9 +25,11 @@
    "id": "595cd38f",
    "metadata": {},
    "source": [
-    "In weather forecasting it is common to run an ensemble of deterministic models. Each ensemble member provides a deterministic forecast. The idea is that the combined ensemble forecasts can be used to estimate a pdf. \n",
+    "In weather forecasting it is common to run an ensemble of deterministic models. Each ensemble member provides a deterministic forecast. The idea is that the combined ensemble forecasts can be used to estimate a PDF, and hence a CDF. \n",
```

Comment:
Sorry, I didn't notice this previously. I think this is the first time the abbreviation "PDF" has been used in this tutorial, and I haven't noticed an earlier explanation of what it stands for. (My guess would be "probability density function", but I don't know if my guess is correct).

Reply:
Correct. Done. However, if you think I need to explain a PDF I might just avoid using the term. Please let me know what you think @Steph-Chong

Reply:
My guess is now that you have added probability density function beforehand, that is enough. (If people aren't sure what it is, now they have the full term, they can google it). But I don't have a background in this area, so if you are concerned, it might be best to get a second opinion from someone with domain expertise.

Reply:
Thanks. I'll leave it. Are you OK to merge this branch or do you want me to assign it to Tennessee?